meta-luneos/conf/distro/include/defaulttunes.inc

```diff
@@ -81,3 +81,5 @@ PACKAGE_EXTRA_ARCHS:append:raspberrypi4-64 = " aarch64-rpi"
 # corei7-64 (Intel Nehalem CPU with 64-bit extensions, MMX, SSE, SSE2, SSE3, SSSE3, SSE4.1, SSE4.2 and POPCNT instruction set support. This tune is recommended for Intel Nehalem and Silvermont (e.g. Bay Trail) CPUs (and beyond)).
 # x86-64-v3 (CPUs with AVX, AVX2, BMI1, BMI2, F16C, FMA, LZCNT, MOVBE, XSAVE. (but not AVX512). See https://www.phoronix.com/news/GCC-11-x86-64-Feature-Levels for details.)
 DEFAULTTUNE:qemux86-64 = "corei7-64"
+
+OLDEST_KERNEL = "3.2.0"
```

Comment:
This part doesn't belong here.

Reply:
oh sorry ! I must have done a git commit -a by mistake. Let me fix that.